README.md

```diff
@@ -20,6 +20,8 @@ access (sudo), you may be prompted for your superuser password.
 5. Execute a destroy/restore cycle by running `./scripts/restore_demo.sh`.
 6. Follow LPBM's logs with `docker-compose logs -f lpbm` and see that the state was restored.
 7. Connect to the DB and see that the latest state was restored.
+8. (Optional) A full backup is created at startup, and once a day (according to config). But a full backup can be triggered any time by running `./scripts/full_backup_demo.sh`. 
```

Comment:
Is the full backup at startup always executed? Not sure this is optimal.

Reply:
No. I meant it's executed when there is journal yet. Thanks for pointing this, I have re-worded to make it clear.